code/__HELPERS/game.dm

```diff
@@ -113,20 +113,31 @@
 	object_to_change.screen_loc = screen_loc
 	return object_to_change
 
+/// Adds an image to a client's `.images`. Useful as a callback.
+/proc/add_image_to_client(image/image_to_remove, client/add_to)
+	add_to?.images += image_to_remove
+
+/// Like add_image_to_client, but will add the image from a list of clients
+/proc/add_image_to_clients(image/image_to_remove, list/show_to)
+	for(var/client/add_to as anything in show_to)
+		add_to?.images += image_to_remove
+
 /// Removes an image from a client's `.images`. Useful as a callback.
 /proc/remove_image_from_client(image/image_to_remove, client/remove_from)
 	remove_from?.images -= image_to_remove
 
-///Like remove_image_from_client, but will remove the image from a list of clients
-/proc/remove_images_from_clients(image/image_to_remove, list/show_to)
-	for(var/client/remove_from in show_to)
-		remove_from.images -= image_to_remove
+/// Like remove_image_from_client, but will remove the image from a list of clients
+/proc/remove_image_from_clients(image/image_to_remove, list/hide_from)
+	for(var/client/remove_from as anything in hide_from)
+		remove_from?.images -= image_to_remove
```

Comment:
I don't understand why we're using ? here, the for() loop should only be affecting clients no?

Reply:
That's fair, I just instinctively added it, images should never be null. Keeping it doesn't cause any harm either, does it?

Reply:
well it kinda removes the point of the ``as anything`` since you'd be checking it anyways

Reply:
Alrighty, removed